File: Java/Events/GetVenuesResult.java
package LTD_API;

import java.util.List;

public class GetVenuesResult
{
    private List<VenueInfo> venues;
    
    public List<VenueInfo> getVenues()
    {
        return venues;
    }
    public void setVenues(List<VenueInfo> venues)
    {
        this.venues = venues;
    }
}

File: PHP/Events/GetVenuesResult.php
<?php
namespace LTD_API;

class GetVenuesResult
{
    //array of VenueInfo
    private $venues;
    
    public function getVenues()
    {
        return $this->venues;
    }
    public function setVenues($venues)
    {
        $this->venues = $venues;
    }
}

